toolset/databases/mysql/mysql.dockerfile

```diff
@@ -1,46 +1,9 @@
-FROM ubuntu:22.04
+FROM mysql:8.0-debian
```

Comment:
Can you switch to `mysql:8-oracle` - it supports AArch64 as well?

Reply:
We don't need AArch64 now.

But more important, is that all the toolset use APT packages, and I don't want to change that.

Reply:
We don't need it for the continuous benchmarking or CI set up by the TechEmpower team, but people might be running the benchmarks in other contexts. For example, the .NET developers at Microsoft are [using](https://devblogs.microsoft.com/dotnet/arm64-performance-improvements-in-dotnet-7/) them to track AArch64 performance.

IMHO since the change is so simple, it is worth making it, so that parts of the community don't need to apply patches downstream. After your changes (which look great BTW) the usage of `apt` is completely transparent, since the Dockerfile does not run a package manager directly.

Anyway, that's my two cents.

Reply:
I think that use Oracle Linux is NOT a good way.

More interesting is start a discussion about using Mariadb, instead or alternative to Mysql.

The Maridb docker image use Ubuntu 22.04, and it supports 4 architectures.

Reply:
> I think that use Oracle Linux is NOT a good way.

From my point of view I don't see a real difference. On the other hand replacing MySQL with MariaDB sounds like a good idea, especially from a platform support perspective.